showcase-app/plugin/identityPlugin.js

```diff
@@ -44,69 +85,72 @@ export default class IdentityWidget {
 
     if (message.type === 'active-identity-response') {
       if (message.data) {
-        let idpOrWebID = message.data.idp;
-
-        if (message.data.webID) {
-          const idps = await this._getIDPsFromWebID(message.data.webID)
-
-          if (idps.length) {
-            idpOrWebID = idps[0];
-          }
-        }
-
-        this.activeIdentity = {
-          ...message.data,
-          idpOrWebID,
-        };
+        // Make the received identity actual
```

Comment:
I don't understand "actual" here.

Reply:
reworded it to make it easy to understand